data/upstream.yaml

```diff
@@ -187,6 +197,13 @@ jabber-roster:
   repo: https://github.com/kparal/jabber-roster
   note: |
     Porting will start if deps are available.
+konversation:
+    links:
+        homepage: https://konversation.kde.org/
+    note:
```

Comment:
This needs to be `note: |` (note the pipe character) so the whole following block is treated as a string.